provisioning/roles/postgis/vars/debian-9.yml

```diff
@@ -0,0 +1 @@
+postgis_package_name: "postgresql-9.6-postgis-2.3"
```

Comment:
I had to rename this to `Debian-9.yml` for this role to work.

Reply:
> My filesystem lacks case-awareness indeed, sorry about that!